// common-constants/index.js
var Showcase   = require('../../components/Showcase/index.jsx'),
    List       = require('../../components/List/index.jsx'),
    Skills     = require('../../components/Skills/index.jsx'),

    LOGOS      = require('../../components/Logos/index.jsx'),
    Javascript = LOGOS.javascript,
    Go         = LOGOS.go,
    Html       = LOGOS.html,
    Css        = LOGOS.css,
    React      = LOGOS.react,
    Node       = LOGOS.node,
    Ruby       = LOGOS.ruby,
    Python     = LOGOS.python,
    Java       = LOGOS.java,
    Mongo      = LOGOS.mongo,
    Postgres   = LOGOS.postgres;

exports.PAGE_TYPES = {
    CONTACT  : 'contact',
    INFO     : 'info',
    MENU     : 'menu',
    PROJECTS : 'projects',
    SLIDE    : 'slide'
}

exports.SLIDE_NAV = {
    PREV : 'prev',
    NEXT : 'next'
}

exports.COMPONENTS = {
    'showcase' : Showcase,
    'list'     : List,
    'skills'   : Skills
}

exports.CLOUDINARY_CLOUD_NAME = "dwkr3rvtr";

exports.LOGO_MAP = {
    'javascript' : {
        title: 'JavaScript',
        component: Javascript
    },
    'go'         : {
        title: 'Go',
        component: Go
    },
    'html'       : {
        title: 'HTML',
        component: Html
    },
    'css'        : {
        title: 'CSS',
        component: Css
    },
    'node'       : {
        title: 'NodeJS',
        component: Node
    },
    'ruby'       : {
        title: 'Ruby',
        component: Ruby
    },
    'python'     : {
        title: 'Python',
        component: Python
    },
    'java'       : {
        title: 'Java',
        component: Java
    },
    'mongo'      : {
        title: 'MongoDB',
        component: Mongo
    },
    'postgres'   : {
        title: 'PostgreSQL',
        component: Postgres
    },
    'react'      : {
        title: 'React',
        component: React
    }
}
